Multisig/UI/UI Library/RemoveSafeButton.swift

```diff
@@ -34,7 +34,7 @@ struct RemoveSafeButton: View {
             ActionSheet(title: Text(""), message: Text("Removing a Safe only removes it from this app. It does not delete the Safe from the blockchain. Funds will not get lost."), buttons: [
                 .destructive(Text("Remove")) {
                     Safe.remove(safe: self.safe)
-                    ViewState.shared.state = .balanaces
+                    App.shared.viewState.state = .balanaces
```

Comment:
@MouazAlzahabi I have emebedded the ViewState into the App so that our app-wide singletons would be accessed in a consistent manner